tools/releaseBuild/template/publish.yml

```diff
@@ -0,0 +1,43 @@
+parameters:
+    stageName: ''
+    environmentName: ''
+    feedCredential: ''
+
+stages:
+- stage: ${{ parameters.stageName }}
+  displayName: Release Microsoft.PowerShell.Pager to '${{ parameters.stageName }}'
+  condition: and(succeeded(), eq(variables['Build.Reason'], 'Manual'), startsWith(variables['Build.SourceBranch'], 'refs/heads/release/'))
+
+  jobs:
+  - deployment: Publish_${{ parameters.stageName }}
+    displayName: Release to Feed
+    pool:
+      name: 1ES
+
+    environment: ${{ parameters.environmentName }}
+    strategy:
+      runOnce:
+        deploy:
+          steps:
+          - download: current
+            artifact: nuget
+
+          - powershell: |
+              Get-ChildItem -Recurse "$(Pipeline.Workspace)/nuget"
+            displayName: Capture downloaded artifact
+
+          - task: NuGetAuthenticate@0
+            condition: ne('${{ parameters.feedUrl }}', '')
+
+          - powershell: |
```

Comment:
consider if we should be user powershell or pwsh

Reply:
Changed to pwsh